### PR TITLE
Refactor author links to below on page (make `.md` table easier to read)

### PR DESCRIPTION
**Pull request by avahoffman**
Hey @cansavvy  - this is a take it or leave it, but markdown tables personally hurt my brain and I wonder if something like this will make it easier to read. It has the added benefit of cutting down on repeated text.

### Purpose/implementation Section

This PR refactors author links embedded in the credits table to links below the table. Makes things cleaner IMO 🙂

#### What changes are being implemented in this Pull Request?

Refactoring, no content change

#### What was your approach?

Replacing links only, no content change

#### What GitHub issue does your pull request address?

None; feature

### Tell potential reviewers what kind of feedback you are soliciting.

Is this something that will help people? Open to ideas for why this is do-not-want!

### New Content Checklist

- [ ] New content/chapter is in an Rmd file with [this kind of format and headers](https://github.com/jhudsl/DaSL_Course_Template_Bookdown/blob/main/02-chapter_of_course.Rmd).

- [ ] New content/chapter contains [Learning Objectives and are in the correct format](https://github.com/jhudsl/DaSL_Course_Template_Bookdown/wiki/Setting-up-images-and-graphics#learning-objectives-formatting).

- [ ] [Bookdown successfully re-renders and any new content files have been added to the _bookdown.yml](https://github.com/jhudsl/DaSL_Course_Template_Bookdown/wiki/Publishing-with-Bookdown).

- [ ] Spell check runs successfully in [Github actions style-n-check](https://github.com/jhudsl/DaSL_Course_Template_Bookdown/wiki/How-to-set-up-and-customize-GitHub-actions-robots#spell-check)).

- [ ] Any newly necessary packages that are needed have been added to the [Dockerfile and image](https://github.com/jhudsl/DaSL_Course_Template_Bookdown/wiki/Using-Docker#adding-packages-to-the-dockerfile).

- [ ] Images are in the [correct format for rendering](https://github.com/jhudsl/DaSL_Course_Template_Bookdown/wiki/Setting-up-images-and-graphics#adding-images-and-graphics-in-text).

- [ ] Every new image has [alt text and is in a Google Slide](https://github.com/jhudsl/DaSL_Course_Template_Bookdown/wiki/Setting-up-images-and-graphics#accessibility).

- [ ] Each slide is described in the notes of the slide so learners relying on a screen reader can access the content. See https://lastcallmedia.com/blog/accessible-comics for more guidance on this.

- [ ] The color palette choices of the slide are contrasted in a way that is friendly to those with color vision deficiencies.
You can check this using [Color Oracle](https://colororacle.org/).

### CODE REVIEW

File: About.Rmd [hunk 1]
```diff
@@ -25,10 +25,10 @@ These credits are based on our [course contributors table guidelines](https://gi
...
 |**Technical**||
 |Course Publishing Engineer(s)| Helped with the code for the technical aspects related to the specific course generation|
-|Template Publishing Engineers|[Candace Savonen](https://www.cansavvy.com/), [Carrie Wright](https://carriewright11.github.io/)|
-|Publishing Maintenance Engineer|[Candace Savonen](https://www.cansavvy.com/)|
-|Technical Publishing Stylists|[Carrie Wright](https://carriewright11.github.io/), [Candace Savonen](https://www.cansavvy.com/)|
-|Package Developers ([Leanbuild](https://github.com/jhudsl/leanbuild))|[John Muschelli](https://johnmuschelli.com/), [Candace Savonen](https://www.cansavvy.com/), [Carrie Wright](https://carriewright11.github.io/)|
+|Template Publishing Engineers|[Candace Savonen], [Carrie Wright]|
+|Publishing Maintenance Engineer|[Candace Savonen]|
+|Technical Publishing Stylists|[Carrie Wright], [Candace Savonen]|
+|Package Developers ([Leanbuild]|[John Muschelli], [Candace Savonen], [Carrie Wright]|
```
Comment (cansavvy) [inline About.Rmd]:
```suggestion
|Package Developers ([Leanbuild])|[John Muschelli], [Candace Savonen], [Carrie Wright]|
```

Comment (cansavvy) [reply About.Rmd]: Otherwise, this seems fine to me! I like reducing extra lines and making it easier for people!

**Review by cansavvy — APPROVED** [review]
Just one paranthesis was lost but otherwise this seems like a good idea to me! (We might need to update the Wiki accordingly. Would you be able to do that, @avahoffman ?

File: About.Rmd (hunk 1 above)
Comment (avahoffman) [reply About.Rmd]: Oops yes! Thanks for catching! I will update the wiki.

**Comment by carriewright11** [comment]
Were we able to check this on Leanpub too? I imagine it will look fine but maybe should check just in case.

**Comment by avahoffman** [comment]
@carriewright11 good call! I have not checked leanpub

**Pull request merged**